plugins/ruby-foreman-docker/debian/control

```diff
@@ -2,12 +2,12 @@ Source: ruby-foreman-docker
 Section: ruby
 Priority: extra
 Maintainer: Pujan Shah <pujan14@gmail.com>
-Build-Depends: debhelper, foreman-compute (>= 1.8.0~rc1), foreman-assets (>= 1.7.0), foreman-sqlite3 (>= 1.8.0~rc1)
+Build-Depends: debhelper, foreman-compute (>= 1.11.0), foreman-assets (>= 1.11.0)
```

Comment:
a) don't remove foreman-sqlite3, which is causing the build error with a missing sqlite3 DB adapter
b) This needs to be `>= 1.11.0~rc1` as it originally was, because `~rc1` is *lower* than 1.11.0